src/go/rpk/pkg/cli/cmd/wasm.go

```diff
@@ -12,14 +12,66 @@ package cmd
 import (
 	"github.com/spf13/afero"
 	"github.com/spf13/cobra"
-	wasm "github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/wasm"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/common"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/cli/cmd/wasm"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/kafka"
 )
 
-func NewWasmCommand(fs afero.Fs) *cobra.Command {
+func NewWasmCommand(fs afero.Fs, mgr config.Manager) *cobra.Command {
+	var (
+		configFile	string
+		brokers		[]string
+	)
+
 	command := &cobra.Command{
 		Use:	"wasm",
-		Short:	"Create and upload inline WASM engine scripts",
+		Short:	"Create, deploy and remove inline WASM engine scripts",
```

Comment:
Just "Deploy and remove [...]", since "deploy" creates it, it's redundant.

Reply:
Done